test/electron/features/feature-flags.feature

```diff
@@ -94,10 +94,10 @@ Scenario: feature flags are attached to unhandled errors in a renderer process
         | Bugsnag-Integrity | {BODY_SHA1}                      |
     And the event contains the following feature flags:
         | featureFlag                | variant    |
-        | from renderer config       |            |
         | from main config 1         | 1234       |
         | from main config 2         |            |
         | from main at runtime       | runtime 1  |
+        | from renderer config       |            |
```

Comment:
One effect of this change is that feature flags added in the main process config will now always appear before those added in the renderer config. This is because all feature flags (and their sort order) are now only stored in the main process.

When an error occurs in the renderer process it receives payload data (including all feature flags) from the main process.

Previously the renderer client stored its own features, and its own separate `client._featuresIndex` object containing only the renderer's feature flags. So when feature flags were merged back across from the main process they would be sorted according to the renderer's `client._featuresIndex`.

Reply:
So this is fixing a bug really — feature flags are now entirely handled in the main process so are always chronologically ordered

Reply:
yep!